tests/Http/PostTest.php

```diff
@@ -76,6 +78,14 @@ public function testCreatingAPostAndSignup()
             ->shouldReceive('userSignup')
             ->shouldReceive('userSignupPost');
 
+        // Mock the GraphQL API calls.
+        $this->mock(GraphQL::class)
+            ->shouldReceive('getUserById');
```

Comment:
Why do we need to mock this here specifically and not in the updated or general event payload tests?

Reply:
Because this post has a `referrer_user_id` set - that's the only time we need to query GraphQL upon post create or update -- to show the post user's display name to the referrer. In Customer.io, we'll have all of the referrer user's info, so the message template would look something like:
```
Hey {{customer.first_name}}!

Guess what? Your friend {{event.user_display_name)) just registered to vote through your online voter registration drive.
```